srcCpp/perftest_cpp.h

```diff
@@ -48,9 +48,23 @@ struct Perftest_ProductVersion_t
   char revision;
 };
 
+struct Perftest_Thread_Priorities {
```

Comment:
New line before the `{`

Reply:
Also, add a small comment mentioning that this class is used to keep the values for the priorities.

Reply:
Done